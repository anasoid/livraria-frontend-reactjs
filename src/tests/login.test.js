import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';
import Login from '../components/login';
import LoginContainer from '../components/login/login-container';


describe('test Login component', () => {
  it('renders without crashing', () => {
    shallow(<Login />);
    const tree = renderer.create(<Login />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders three LoginContainer component', () => {
    const wrapper = shallow(<Login />);
    expect(wrapper.find(LoginContainer)).toExist();
  });

});