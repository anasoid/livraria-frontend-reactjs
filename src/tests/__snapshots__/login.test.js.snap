// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test Login component renders without crashing 1`] = `
<div>
  <span
    id="messages"
  />
  <form
    onSubmit={[Function]}
  >
    <fieldset>
      <legend>
        Login
      </legend>
      <label
        htmlFor="loginEmail"
      >
        Email:
        <span>
          *
        </span>
      </label>
      <input
        id="loginEmail"
        name="email"
        type="email"
      />
      <br />
      <label
        htmlFor="loginSenha"
      >
        Senha:
        <span>
          *
        </span>
      </label>
      <input
        id="loginSenha"
        name="senha"
        type="password"
      />
      <br />
      <button
        type="submit"
      >
        Efetue Login
      </button>
    </fieldset>
  </form>
</div>
`;
